modules/tensor_mechanics/src/materials/crystal_plasticity/FiniteStrainUObasedCP.C

```diff
@@ -447,7 +445,16 @@ FiniteStrainUObasedCP::solveStress()
   if (iter >= _maxiter)
   {
 #ifdef DEBUG
```

Comment:
this should be `#ifndef NDEBUG` ... yeah, I know. But that's how we enable assertions for example. Otherwise it won't show up in `devel` mode